SPCTR-3463: Fix - FAQ block content default padding issue in editor. (#3575)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Forms - Added note for Forms block duplicate name fields to have distinct labels.
* Fix: Global - Resolved an issue in which the Spectra blocks were unable to be previewed in the widget editor customizer.
* Fix: Buttons - Resolved an issue in which the margins added to the button group would get overridden by child button margins.

> * Fix: FAQ - Resolved an issue in which the default content padding is not same in editor as in frontend.

Reviewer: ```suggestion
* Fix: FAQ - Resolved an issue in which the default content padding would appear differently in the editor and the front-end.
```

Author: Done.
